scripts/run_smoke_tests.py

```
@@ -959,6 +959,15 @@ def verify_mineru_agent_helper_guards() -> None:
     if not probe_materials.mojibake_metrics("\u0080" * 200)["mojibake_suspect"]:
         raise AssertionError("Expected repeated C1-control mojibake sample to be detected")
 
+    # Latin-1 Supplement / punctuation mojibake (e.g. no-ToUnicode SimSun subset PDFs)
+    # must be detected even when C1-control ratio alone is below the threshold.
+    latin_mojibake_sample = "Æµ‚SØ‡‰K" * 100
```

Ruff (0.16.1)

[warning] String contains ambiguous `‚` (SINGLE LOW-9 QUOTATION MARK). Did you mean `,` (COMMA)?

(RUF001)